### PR TITLE
feat: update SAP BTP Icons periodically

### PR DESCRIPTION
**Pull request by cschuerings**
This PR adds a GitHub Action, which periodically (once per week) updates the BTP icons by executing the script from #1.

Here is a successful run:
https://github.com/cschuerings/sap_btp_icons_drawio_lib/actions/runs/4415557702

And the resulting commit:
https://github.com/cschuerings/sap_btp_icons_drawio_lib/tree/test-update

### CODE REVIEW

File: .github/workflows/update-icons.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - name: "Install NodeJS dependencies"
+        run: |
+          npm ci
+      - name: "Read new SAP BTP metadata and update icons"
+        run: |
+          npm start
+          git add -A
+          git config user.email "btpiconbot@users.noreply.github.com"
+          git config user.name "BTP Icon Bot"
+          git diff --quiet && git diff --staged --quiet || git commit -m "[bot] $(date '+%Y-%m-%d %H:%M:%S'): updated BTP icons"
```
Comment (mauriciolauffer) [inline .github/workflows/update-icons.yml]: I'd remove the date from the message, the commit itself has a timestamp.

**Comment by mauriciolauffer** [comment]
I loved it! It's been in my backlog forever, thanks for implementing it 🥇 
I left a comment regarding the commit message, other than that, LGTM!

File: .github/workflows/update-icons.yml (hunk 1 above)
Comment (cschuerings) [reply .github/workflows/update-icons.yml]: Sure, I removed the timestamp from the message.

**Review by mauriciolauffer — APPROVED** [review]
LGTM!

**Comment by mauriciolauffer** [comment]
@rsletta I believe we need your help merging this PR  :)

**Comment by rsletta** [comment]
Thanks for your contribution @cschuerings, that is a great addition!

**Pull request merged**